Changelog.md

```diff
@@ -10,6 +10,7 @@
 - Add visual indicator on a per-assignment basis for used grace credits (#7226)
 - Change default disabled text area colour to ligher grey on light mode (#7269)
 - Implement a limit on the file size rendered by the submission viewer (#7273)
+- Give instructors the ability to delete a TA from the Users Graders Tab (#7304)
```

Comment:
Please do a merge from master, as we just made a new release. Then, you'll need to *move* this entry from where it currently is (it'll be "merged" into the new 2.6.0 section) into the new, smaller [unreleased] section above 2.6.0.